Formula/duplicity.rb

```diff
@@ -324,10 +325,46 @@ def install
     # OSX doesn't provide a /usr/bin/python2. Upstream has been notified but
     # cannot fix the issue. See:
     #   https://github.com/Homebrew/homebrew/pull/34165#discussion_r22342214
-    inreplace "#{libexec}/bin/duplicity", "python2", "python"
+    inreplace libexec/"bin/duplicity", "python2", "python"
+
+    # We prefer gpg2, so set a default. This can still be overriden by
+    # passing the `--gpg-binary=` command line flag.
+    inreplace libexec/"lib/python2.7/site-packages/duplicity/globals.py",
+              "gpg_binary = None", "gpg_binary = 'gpg2'"
```

Comment:
I tend to lean towards thinking how we name `gpg`s is a packager-specific quirk rather than an upstream problem, in the same way we've had to `inreplace` a few `python2` hashbangs out for OS X.

Sadly this is only going to get a little more tricky, Debian are moving towards `gpg1` for `gnupg` which I think we'll want to mimic eventually, and installing `gnupg2` as `gpg2` with a symlink to `gpg`. I think that's a sane system but I'm pretty reluctant to hurdle Debian.